src/main/features/response/routes/reject-part-of-claim.ts

```diff
@@ -0,0 +1,39 @@
+import * as express from 'express'
+
+import { Paths } from 'response/paths'
+
+import { FormValidator } from 'forms/validation/formValidator'
+import { Form } from 'forms/form'
+
+import { RejectPartOfClaim } from 'response/form/models/rejectPartOfClaim'
+import { ResponseDraftMiddleware } from 'response/draft/responseDraftMiddleware'
+import { ErrorHandling } from 'common/errorHandling'
+import User from 'idam/user'
+
+function renderView (form: Form<RejectPartOfClaim>, res: express.Response) {
+  res.render(Paths.defenceRejectPartOfClaimPage.associatedView, {
+    form: form
+  })
+}
+
+export default express.Router()
+  .get(Paths.defenceRejectPartOfClaimPage.uri, ErrorHandling.apply(async (req: express.Request, res: express.Response, next: express.NextFunction) => {
```

Comment:
That route has to be protected by guard so it is impossible to access it unless somebody selected `OWE_SOME_PAID_NONE` on the response type page.

Remember to cover both GET and POST handlers and reflect that fact in tests as well.